### ISSUE TITLE
Request: merge with Platform.enviroment 

### ISSUE THREAD

**Issue by skyred** [java-james/flutter_dotenv#11]
As the original package does:

```
var _env = new Map<String, String>.from(Platform.environment);

/// A copy of [Platform.environment](dart:io) including variables loaded at runtime from a file.
Map<String, String> get env => _env;

/// Overwrite [env] with a new writable copy of [Platform.environment](dart:io).
Map clean() => _env = new Map.from(Platform.environment);
```

Please merge in Platform.enviroment variables, which are useful too. Then this flutter plugin will provide application one method to handle all env variables.

**Comment by java-james** [comment]
@skyred Thanks for the suggestion I will look into this.

**Comment by java-james** [comment]
New published 3.0.0 now optionally merges with Platform.Environment.
